src/chi/components/data-table/data-table.scss

```diff
@@ -871,3 +879,38 @@ $bg-url: $bg-url + ' fill="%230262b9" fill-rule="evenodd"/%3E%3C/svg%3E';
     }
   }
 }
+
+@media print {
+  .chi-data-table {
+    .default-view {
+      display: none;
+    }
+
+    .print-only {
+      display: block;
+
+      .chi-table {
+        transform-style: preserve-3d;
+
+        thead {
+          display: table-row-group;
+        }
+
+        tbody {
+          tr {
+            td::before,
+            td::after {
+              content: '';
+              height: 0.25rem;
+              display: block;
+            }
+          }
+        }
+
+        tfoot {
+          display: table-row-group;
+        }
+      }
+    }
+  }
```

Comment:
I'm not sure if all this is necessary having generic screen and print visibility classes.

Reply:
what do you mean sorry? The display: table-row-group; and transform-style: preserve-3d; are used to prevent header/footer repetition which overlaps with the body rows, and the height and content for td is used to avoid cell break between pages